crates/sui-graphql-rpc/src/types/balance_change.rs

```diff
@@ -9,8 +9,10 @@ use super::{big_int::BigInt, move_type::MoveType, owner::Owner, sui_address::Sui
 use crate::error::Error;
 
 pub(crate) struct BalanceChange {
-    // TODO: Move this type into indexer crates, rather than JSON-RPC.
     stored: StoredBalanceChange,
+    // BalanceChange is only available when a tx has been indexed, so it will always have a
+    // checkpoint_sequence_number
+    checkpoint_viewed_at: u64,
```

Comment:
This comment no longer applies, right? Because it's no longer about whether the `tx` has a checkpoint sequence number or not, it's about what checkpoint this information was fetched from (which means this should also be an `Option<u64>`.

Reply:
Yes good point